### DISCUSSION TITLE
Manual order creation

### DISCUSSION THREAD

**Discussion by BE-CH** [medusajs/medusa#9829] [Feature Requests]
Add manual order creation to the admin panel. This would allow for orders that were created over the phone or anywhere else that is not on the webshop.

![image](https://github.com/user-attachments/assets/068d10b2-83eb-419e-97bf-1edb31c54ab5)

**Comment by radoslaw-sz** [comment]
hi @BE-CH  - I am not sure if you are asking only about the button or the full implementation, but you can implement it by using widgets and put it before the Order list

**Reply by BE-CH** [reply]
I am asking for the full implementation.

**Comment by dimitrio-m** [comment]
This was available in V1 and I consider it a must for ecommerce.

**Comment by eugenepro2** [comment]
@kasperkristensen I think it would be a useful feature. If I implement it and prepare a PR, would it align with the roadmap for the Medusa team?

**Motivation:** Sometimes user create order by phone or messenger. Manger should create a new order ourselves, but at this time, we cannot do it inside Medusa Admin, like it works for creating a product or category.

**Expected result:** A new form in Order domain, to create a new order.

**Reply by BE-CH** [reply]
@eugenepro2 I totally agree! I get tons of orders by phone and this feature would be so great. 

Thanks for your initiative. 😄

**Reply by BE-CH** [reply]
@eugenepro2 Did you ever have time to look at this? 😊

I am considering trying to open a PR myself.

**Reply by eugenepro2** [reply]
Medusa announced [draft order](https://medusajs.com/blog/announcing-draft-orders/). 

> The Draft Order plugin is ready today for Medusa Cloud users, with an open-source release planned later this year to explore Medusa Cloud and start building custom commerce solutions faster.

I just wait this feature. @BE-CH

**Comment by dimitrio-m** [comment]
Hello! Are there any updates regarding this?
